From 0ff0b3ef70525480da8ab88d409fa9e195556cef Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 15 Sep 2014 15:59:38 +0800
Subject: [PATCH] ENGR00331450-11: ARM: dts: imx6dl-sabresd-pf200: use
 ldo-enable mode

For pfuze200, we have to use ldo-enable mode, because VDDARM_IN and VDDSOC_IN
share the same regulator.
Add GPL license head.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6dl-sabresd-pf200.dts | 12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd-pf200.dts b/arch/arm/boot/dts/imx6dl-sabresd-pf200.dts
index 7b48033..6451ca6 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd-pf200.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd-pf200.dts
@@ -1,5 +1,13 @@
-#include "imx6dl-sabresd.dts"
+/*
+ * Copyright (C) 2014 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
 
-&pmic{
+#include "imx6dl-sabresd-ldo.dts"
+
+&pmic {
 	compatible = "fsl,pfuze200";
 };
-- 
1.8.0

